src/components/ProductListPage/CheckBoxList/CheckBoxList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CheckBox from './CheckBox';
+
+const CheckBoxList = ({ itemsList, category, checkCheckbox }) => {
+  const renderCheckboxes = itemsList.map(({ item }) => (
+    <li key={item}>
+      <CheckBox
+        itemName={item}
+        checkCheckbox={checkCheckbox}
+        category={category}
+      />
+    </li>
```

Comment:
Well, in this case, you can put `<li>` inside of `<Checkbox>` instead of `<dvi>`